### ISSUE TITLE
vfmt fn type regression

### ISSUE THREAD

**Issue by serkonda7** [vlang/v#6756]
**V version:** 
**OS:** 

**What did you do?**
```v
pub fn easing(easingtype EasingType) EasingFunction {
...
}
```

**What did you see instead?**
```v
pub fn easing(easingtype EasingType) fn (f64) f64 {
...
}
```

**Comment by serkonda7** [comment]
Duplicate of a small part of #6755

**Comment by spytheman** [comment]
Fixed in d34c5b7
